src/Sentry/PlatformAbstractions/Runtime.cs

```diff
@@ -0,0 +1,138 @@
+using System;
+
+namespace Sentry.PlatformAbstractions
+{
+    /// <summary>
+    /// Details of the runtime
+    /// </summary>
+    /// <inheritdoc />
+    public class Runtime : IEquatable<Runtime>
+    {
+        private static Runtime? _runtime;
+        /// <summary>
+        /// Gets the current runtime
+        /// </summary>
+        /// <value>
+        /// The current runtime.
+        /// </value>
+        public static Runtime Current => _runtime ??= RuntimeInfo.GetRuntime();
+        /// <summary>
+        /// The name of the runtime
+        /// </summary>
+        /// <example>
+        /// .NET Framework, .NET Native, Mono
+        /// </example>
+        public string? Name { get; internal set; }
+        /// <summary>
+        /// The version of the runtime
+        /// </summary>
+        /// <example>
+        /// 4.7.2633.0
+        /// </example>
+        public string? Version { get; internal set; }
+#if NETFX
+        /// <summary>
+        /// The .NET Framework installation which is running the process
+        /// </summary>
+        /// <value>
+        /// The framework installation or null if not running .NET Framework
+        /// </value>
+        public FrameworkInstallation FrameworkInstallation { get; internal set; }
```

Comment:
This class implements `IEquatable<T>` but has mutable properties. Could be dangerous if used in `Dictionary`.

Reply:
True. Removed the interface